src/components/PostList/PostList.jsx

```diff
@@ -1 +1,10 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo';
+import './PostList.scss';
+
+export const PostList = ({ posts }) => (
```

Comment:
Ensure that the `posts` prop is correctly passed to the `PostList` component. This prop should be an array of post objects.